libjpeg_addons/include/libjpg_addons.h

```diff
@@ -0,0 +1,33 @@
+#ifndef __LIBJPG_H__
+#define __LIBJPG_H__
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+#include <tamtypes.h>
```

Comment:
Why is tamypes.h included? Only a header for the type "FILE" is needed I think.

Reply:
Fixed